input_cloud_topic: "/map_nodelet/pointcloud"
input_image_topic: "/map_nodelet/scene_cloud_registered"
